app/services/auctions_closed_within_last_twenty_four_hours_finder.rb

```diff
@@ -0,0 +1,9 @@
+class AuctionsClosedWithinLastTwentyFourHoursFinder
```

Comment:
that's a great idea! I think I created that other service class before I was familiar with the codebase so I didn't know about our query class